<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Show_info_Watch</class>
 <widget class="QDialog" name="Show_info_Watch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
color: rgb(0, 85, 127);</string>
   </property>
   <property name="text">
    <string>INFORMATION (SMARTWATCH)</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>231</width>
     <height>415</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Brand:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="des_lineEdit_2"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Price:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="capacity_2"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Bluetooth:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="bluetooth_lineEdit_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>Display:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_31">
      <property name="text">
       <string>Capacity:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_32">
      <property name="text">
       <string>Srceen size:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_33">
      <property name="text">
       <string>Wifi:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="case_size_2"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Case size:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="wifi_lineEdit_2"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="brand_lineEdit_2"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="model_lineEdit_2"/>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="screen_size_2"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="price_lineEdit_2"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Quantity:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="display_lineEdit_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="quantity_lineEdit_2"/>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string>Brightness:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="brightness_lineEdit_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>531</width>
     <height>333</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="3" column="1">
     <widget class="QLineEdit" name="weight_lineEdit_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_35">
      <property name="text">
       <string>Dimension:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="water_resistance_2"/>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_36">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="battery_lineEdit_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_37">
      <property name="text">
       <string>Weight:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_38">
      <property name="text">
       <string>Battery:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QCheckBox" name="has_sim_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>has sim</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="dimension2_lineEdit_2"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="dimension1_lineEdit_2"/>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="has_mic_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>has microphone</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_39">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_40">
      <property name="text">
       <string>Water resistance:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="has_speaker_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>has speaker</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="dimension3_lineEdit_2"/>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="has_blood_O2_sensor_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>has Blood O2 sensor</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="has_gps_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>has gps</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="has_heartrate_monitering_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>heartrate monitoring</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_41">
      <property name="text">
       <string>meter</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_42">
      <property name="text">
       <string>mAh</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
